std_msgs/Header header

bool[] contact_states
geometry_msgs/Vector3[] force_vec
geometry_msgs/Vector3[] vel_vec
geometry_msgs/PoseWithCovariance[] pose_vec
geometry_msgs/TwistWithCovariance[] twist_vec
